lib/ActivityStreams.js

```diff
@@ -266,8 +274,12 @@ ActivityStreams.prototype = {
   },
 
   /**
-   * Removes a worker and calls callback if defined
+   * Logs telemetry event
    */
+  _logPerformance(tag, worker, data) {
```

Comment:
I would like to keep this indirection as we may need some other logic executed before we sent an event to the perfMeter.  For example, I am customary using this API to place debug output and see which logs and when are generated by the application rather then the PerfMeter.